functions/Test-DbaLastBackup.ps1

```diff
@@ -259,7 +283,56 @@ Restores data and log files to alternative locations and only restores databases
 					}
 					
 					$lastbackup = Get-DbaBackupHistory -SqlServer $sourceserver -Databases $dbname -LastFull -IgnoreCopyOnly:$ignorecopyonly
-										
+					
+					if($lastbackup[0].Path.StartsWith('\\') -eq $false -and $CopyDestination) 
+					{
+						Write-Message -Level Verbose -Message "Copying backup to destination server"
+						if($copyPath.StartsWith('\\') -eq $false)
+						{
+							Write-Message -Level Verbose -Message "Destination backup directory is not UNC and source does not match destination."
+							$fileexists = "Skipped"
+							$restoreresult = "Destination backup not on shared location."
+							$dbccresult = "Skipped"
+						}
+						elseif ((Test-SqlPath -SqlServer $destserver -path $copyPath)) 
+						{
+							if((Test-SqlPath -SqlServer $destserver -path ("{0}\{1}" -f $copyPath,$Prefix)) -eq $false)
+							{
+								try 
+								{
+									New-Item -Path ("{0}\{1}" -f $copyPath,$Prefix) -ItemType Directory | Out-Null
```

Comment:
Change to:
`$null = New-Item -Path ("{0}\{1}" -f $copyPath,$Prefix) -ItemType Directory -ErrorAction Stop`
No pipeline = better performance
Without the erroraction parameter, try/catch won't do a thing